buildbot/master/files/config/factories.py

```diff
@@ -176,9 +176,11 @@ def make_step(self, command):
 
         command = command.split(' ')
 
-        # Add `bash -l` before every command on Windows builders
-        bash_args = ["bash", "-l"] if self.is_windows else []
-        step_kwargs['command'] = bash_args + command
+        # Add `bash -l -c` and export Windows Python before every command
+        # on Windows builders
+        bash_args = "export PATH=/c/python27:/c/python27/scripts:$PATH; "
```

Comment:
Alternately, @larsbergstrom is there a `.profile` or `.bash_profile` that would reset the `$PATH` when we run bash?